components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -1489,7 +1489,7 @@ document.observe('xwiki:dom:loaded', function()
     initialize : function(dataTable) {
       /** Measurement evaluation service */
       this.measurementsService = new XWiki.Document('PatientToolsService', 'PhenoTips').getURL('get');
-      this.solarService = new XWiki.Document('SolrService', 'PhenoTips').getURL("get");
+      this.solrService = XWiki.contextPath + "/rest/vocabularies/hpo/suggest";
```

Comment:
actually, since it is technically does not have to be solr-based any more wouldn't it be better to rename it to `hpoSuggestService` (or smth similar), which it is? I mean, from the client code point of view it does not mater what technology we use at the back-end